python/casa_distro/environment.py

```diff
@@ -826,12 +827,17 @@ def tests(self, test_config, dev_config):
                     command = command.replace('/casa/host/build/bin/bv_env',
                                               '/casa/install/bin/bv_env')
                 result, output = self.call_output([self.casa_distro,
-                    'run',
-                    'name={0}'.format(test_config['name']),
-                    'env=BRAINVISA_TEST_RUN_DATA_DIR=/casa/host/tests/test,'
-                    'BRAINVISA_TEST_REF_DATA_DIR=/casa/host/tests/ref',
-                    '--',
-                    'sh', '-c', command])
+                                                   'run',
+                                                   'name={0}'.format(
+                                                       test_config['name']),
+                                                   'env=BRAINVISA_'
+                                                   'TEST_RUN_DATA_DIR='
```

Comment:
Cannot unindent my code and line must be 79 characters max. This is much, much more readable. Thank you PEP 8 !